### PR TITLE
Updated Windows dev environ setup instructions

### PR DESCRIPTION
**Pull request by itsjoshthedeveloper**
added instructions to clone repository in linux filesystem. this does several things:
- [x] allow for React hot reloading so we don't need to restart the frontend server every time
- [x] remove the step for having to change CRLF to LF since it's already LF
- [x] 3x faster docker-compose ups

### CODE REVIEW

**Comment by hsheth2** [comment]
This is awesome - should make windows dev much more streamlined!

Added @nsnave since I can't really review it without a windows machine

File: docs/getting-started.md [hunk 1]
```diff
@@ -7,80 +7,66 @@ Thanks for taking the initiative to get started with CourseTable! Here are a few
...
-
-    Once installed, open VSCode and install the following extensions:
+<details>
+  <summary><strong>Initial setup for Windows only</strong></summary>
 
-    - [Remote - WSL](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl) (only needed for Windows)
-    - [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
-    - [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
-    - [GraphQL](https://marketplace.visualstudio.com/items?itemName=GraphQL.vscode-graphql)
+Cause Windows is a special little baby, there's some things we got to do to get this working correctly.
```
Comment (nsnave) [inline docs/getting-started.md:15]: lol

**Comment by itsjoshthedeveloper** [comment]
can i go ahead and merge? @nsnave

**Review by nsnave — APPROVED** [review]
good job, you can merge

**Pull request merged**